template/vtex/tsconfig.json

```diff
@@ -1,15 +1,8 @@
 {
   "extends": "@vtex/tsconfig",
   "compilerOptions": {
-    "noEmitOnError": false,
-    "typeRoots": ["node_modules/@types"],
-    "types": ["node", "jest", "graphql"],
-    "module": "esnext",
-    "moduleResolution": "node",
-    "target": "es2017",
     "rootDirs": ["src", "stories"],
     "baseUrl": "src",
   },
-  "include": ["src"],
-  "exclude": ["node_modules", "dist", "example"]
```

Comment:
this line must be maintained, just rename `example` to `stories`

Reply:
Resolved